sources/packages/backend/apps/db-migrations/src/sql/Applications/Update-comment-location-id.sql

```diff
@@ -0,0 +1 @@
+COMMENT ON COLUMN sims.applications.location_id IS 'References the location related to the application at the submission. From the location it is possible to get the institution for example.';
```

Comment:
Thanks for adjusting the comment.

As a suggestion, it could be simplified as `References the institution location of the application.`